src/patternfly/components/Form/examples/form-select-input-example.hbs

```diff
@@ -1,35 +1,41 @@
 {{#> form}}
   {{#> form-group}}
     {{#> form-label for="select1"}}Select input{{/form-label}}
-    {{#> form-control}}
-      {{#> form-select id="select1" name="example-select1"}}
-        {{#> form-select-option value="Option 1"}}The first option{{/form-select-option}}
-        {{#> form-select-option value="Option 2"}}The second option{{/form-select-option}}
-      {{/form-select}}
-    {{/form-control}}
+    {{#> form-controls controlType="select" id="select1" name="selectexample1"}}
+      <option value="" selected>Please choose</option>
```

Comment:
What do you think about removing the `value` from the "please choose" type of option, and setting it to `disabled`, so that you have to choose something?

Reply:
that works for me